sbncode/NuMuSelection/Algorithms/FiducialVolume.h

```diff
@@ -0,0 +1,88 @@
+#ifndef FIDUCIALVOLUME_H
+#define FIDUCIALVOLUME_H
+/**
+ * \file FiducialVolume.h
+ *
+ * \ingroup UBXSec
+ * 
+ * \brief Class def header for a class FiducialVolume
+ *
+ * @author Marco Del Tutto
+ *
+ * Taken from Marco Del Tutto's UBXSec analysis
```

Comment:
The licensing situation in LArSoft is unfortunately a mess as far as I can tell. Could you use the code in UBXsec via uboonecode libraries, or implement FV cuts in your own code?

Reply:
Also removed this one, since we only use AABox stuff in the sbnanalysis code.